testdata/dnn/darknet/generate_darknet_models.py

```diff
@@ -37,4 +37,4 @@ def genTestData(name, inpShape, outShape):
 genTestData('route_multi', [1, 6, 3, 6], [1, 4, 2, 3])
 genTestData('relu', [1, 2, 3, 4], [1, 2, 3, 4])
 genTestData('sam', [1, 2, 3, 4], [1, 4, 3, 4])
-
+genTestData('crop', [1, 3, 5, 6], [1, 3, 3, 3])
```

Comment:
`crop_width` and `crop_height` are output dimensions. So should be `genTestData('crop', [1, 3, 5, 6], [1, 3, 3, 2])`

Reply:
You want me to generate files for `genTestData('crop', [1, 3, 5, 6], [1, 3, 3, 2])`?

Reply:
Yes, because correct output size is 1x3x3x2

Reply:
Okay. I will do it right away.